docs/modules/deploy/pages/versioning-compatibility.adoc

```diff
@@ -171,15 +162,14 @@ Hazelcast Platform has been tested against the following JVMs. It may run on oth
 |Red Hat build of OpenJDK 11
 |✓
 
-|Microsoft Build of OpenJDK 11
-|✓
-
```

Comment:
Why is this gone? See the complete list here:
https://docs.google.com/document/d/1BzmPiXuC9MyJ4WDwjvt1f0dJKzWoudT_rqT0TKUPGt0/edit#heading=h.vhojv0k3bqco

Reply:
Same as above.